src/components/dialogs/commons/grid-item.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { PropsWithChildren } from 'react';
+import { Grid } from '@mui/material';
+
+interface GridItemProps extends PropsWithChildren {
```

Comment:
```suggestion
export interface GridItemProps extends PropsWithChildren {
```
It is better to export also props to someone need to extends